app/assets/stylesheets/wpcc/section/_details.scss

```diff
@@ -13,6 +13,11 @@
   }
 }
 
+.details__error-summary,
+.details__row--group {
+  @include column(12);
+}
+
```

Comment:
Nope just in this project, its because the input fields are in a 30% width container (at full screen size), and the margin is just over 1% of that, in order for the error summary to be aligned with the errors on the fields we need to apply a style which applies the same margin at each screen size, and only for the errors.  That probably makes no sense but it is the reason :)